### ISSUE TITLE
Multimeter issues

### ISSUE THREAD

**Issue by Andy1910** [snootched/cb-lcars#10]
i don't know what i am doing wrong each time i try to use the multimeter i get an error saying custom element does not exist:CBlcars-my-sliders-V2 even though my card bundle is installed and working

**Comment by snootched** [comment]
Make sure you've removed any remnants of the old version tempates from your dashboard file (ie. if you copied the previous yaml templates from github and pasted them into your dashboard.)
That custom card isn't used, and you just need the original my-slider cards from HACS.
To test - you can try the latest alpha build from here, and use on a new blank dashboard.

**Comment by adejong007** [comment]
I've been running into the same issue. It only happens with the slider mode. I'm using a new dashboard and I've refreshed the browser. HACS shows I'm using CB-LCARS v2024.12.4 and My Cards Bundle v1.0..

![image](https://github.com/user-attachments/assets/0560542d-360e-4f45-a9d4-cddbd7c69b46)

This was made using the UI, but here's the yaml.

```
    sections:
      - type: grid
        cards:
          - type: heading
            heading: New section
          - type: custom:cb-lcars-multimeter-card
            enable_resize_observer: true
            variables:
              entity_match_header: false
              entity_match_slider: false
              entity_match_gauge: false
              entity_match_slider_start: false
              entity_match_slider_end: false
              _mode: slider
              _vertical: true
              _slider_mode: brightness
              card:
                border:
                  left:
                    size: 120
                  top:
                    size: 10
              gauge:
                sub_meter:
                  show_sub_meter: true
              entity: sensor.kitchen_temperature
            tap_action:
              action: toggle
            double_tap_action:
              action: none
            hold_action:
              action: none
            show_label: false
          - type: custom:cb-lcars-multimeter-card
            enable_resize_observer: true
            variables:
              entity_match_header: false
              entity_match_slider: false
              entity_match_gauge: false
              entity_match_slider_start: false
              entity_match_slider_end: false
              _mode: gauge
              _vertical: true
              _slider_mode: brightness
              card:
                border:
                  left:
                    size: 120
                  top:
                    size: 10
              gauge:
                sub_meter:
                  show_sub_meter: true
              entity: sensor.kitchen_temperature
            tap_action:
              action: toggle
            double_tap_action:
              action: none
            hold_action:
              action: none
            show_label: false
          - type: custom:cb-lcars-multimeter-card
            enable_resize_observer: true
            variables:
              entity_match_header: false
              entity_match_slider: false
              entity_match_gauge: false
              entity_match_slider_start: false
              entity_match_slider_end: false
              _mode: slider
              _vertical: false
              _slider_mode: brightness
              card:
                border:
                  left:
                    size: 120
                  top:
                    size: 10
              gauge:
                sub_meter:
                  show_sub_meter: true
              entity: sensor.kitchen_temperature
            tap_action:
              action: toggle
            double_tap_action:
              action: none
            hold_action:
              action: none
            show_label: false
          - type: custom:cb-lcars-multimeter-card
            enable_resize_observer: true
            variables:
              entity_match_header: false
              entity_match_slider: false
              entity_match_gauge: false
              entity_match_slider_start: false
              entity_match_slider_end: false
              _mode: gauge
              _vertical: false
              _slider_mode: brightness
              card:
                border:
                  left:
                    size: 120
                  top:
                    size: 10
              gauge:
                sub_meter:
                  show_sub_meter: true
              entity: sensor.kitchen_temperature
            tap_action:
              action: toggle
            double_tap_action:
              action: none
            hold_action:
              action: none
            show_label: false
```

**Comment by snootched** [comment]
Perhaps try incognito/inprivate to verify.  I took your code (just changed entity name to one of mine) and it renders:

![image](https://github.com/user-attachments/assets/7ebc85f2-d7ee-4c9d-99b5-ac483f5be02e)

(I need to update the slider mode if the enityt provided is not a controllable type..) but it doesn't reference the old custom element not existing.

if it persists, can you run in the dev tools console:

`window.cblcars` and see if there is a `tainedCards` key.
![image](https://github.com/user-attachments/assets/dc0a0573-eccf-4d32-8ab9-c27ec7cc4f6a)

**Comment by adejong007** [comment]
Oh I see. I didn't realize the error had switched from the "custom element does not exist slider-v2" error to the "entity has to be one of the following" error. 

I guess this issue is closed for me as of v2024.12.4. I'll keep an eye out for a release with the updated slider for non-controllable entities :) Thanks for sharing these cards!

**Comment by snootched** [comment]
Great - glad that is solved for you.  The slider updates should be in the next release.
